lib/scimitar_schema_extension.rb

```diff
@@ -48,6 +48,49 @@ class OpenProjectNameComplexType < Scimitar::ComplexTypes::Base
 end
 
 module ScimitarSchemaExtension
+  AUTHENTICATION_SCHEMES = [
+    Scimitar::AuthenticationScheme.new(
+      type: "oauth2",
+      name: "OAuth2",
+      description: OpenProject::Static::Links.links[:sysadmin_docs][:scim_static_access_token_authentication_method][:href]
+    ),
+    Scimitar::AuthenticationScheme.new(
+      type: "oauthbearertoken",
+      name: "OAuth Bearer Token",
+      description: OpenProject::Static::Links.links[:sysadmin_docs][:scim_oauth2_client_credentials_authentication_method][:href]
+    ),
+    Scimitar::AuthenticationScheme.new(
+      type: "oidcjwt",
+      name: "OpenID Provider JWT",
+      description: OpenProject::Static::Links.links[:sysadmin_docs][:scim_jwt_authetication_method][:href]
+    )
+  ].freeze
+
+  class LimitedServiceProviderConfiguration
```

Comment:
:yellow_circle: I guess I'd also prefer more files for all these classes in this file. I noticed how I was irritated when I saw the `super` call below and didn't quite know where to scroll to find what "super" means.